sql_generators/glean_usage/templates/usage_reporting_clients_last_seen_v1.query.sql

```diff
@@ -0,0 +1,54 @@
+{{ header }}
+
+WITH _current AS (
+  SELECT
+    usage_profile_id,
+    -- In this raw table, we capture the history of activity over the past
+    -- 28 days for each usage criterion as a single 64-bit integer. The
+    -- rightmost bit in 'days_since_seen' represents whether the user sent a
+    -- usage_reporting ping in the submission_date and similarly, the rightmost bit in
+    -- days_active_bits represents whether the user counts as active on that date.
+    CAST(TRUE AS INT64) AS days_seen_bits,
+    CAST(TRUE AS INT64) & CAST(is_active AS INT64) AS days_active_bits,
+    udf.days_since_created_profile_as_28_bits(
+      DATE_DIFF(submission_date, first_run_date, DAY)
+    ) AS days_created_profile_bits,
```

Comment:
@irrationalagent Is the `days_created_profile_bits` field important? Right now it seems `first_run_date` is not available in the new ping. Is this intended?